Woof/Woof/View/ReusableViews/CopyToClipboardLabel.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+/// A label for user interface items, consisting of an icon with two docs and with title `Copy`.
+struct CopyToClipboardLabel: View {
+    // MARK: - Internal interface
+
+    var body: some View {
+        Label {
+            Text(labelText)
+        } icon: {
+            Image(systemName: String.IconName.doc)
+        }
+    }
+
+    // MARK: - Private interface
+
+    private let labelText = "Copy"
```

Comment:
Why do we need to create private property and use it only once in our code?

Reply:
To convey intention more clearly and avoid magic strings.

Reply:
How it helps avoid the magic string, it still exists in the same file, you just move it below